<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Controller" name="paintCon" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>711</width>
      <height>751</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="methodTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>720</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Method：</string>
     </property>
    </widget>
    <widget class="QLabel" name="targetTitle">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>720</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Target：</string>
     </property>
    </widget>
    <widget class="QLabel" name="methodLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>720</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="targetLabel">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>710</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>170</y>
      <width>141</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="CCDsingle">
       <property name="text">
        <string>单次CCD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CCD">
       <property name="text">
        <string>循环坐标下降法</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Jacobi">
       <property name="text">
        <string>逆向雅克比法</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start/Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="trace">
       <property name="text">
        <string>轨迹显示</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="circletrace">
       <property name="text">
        <string>弧形运动</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Controller</class>
   <extends>QWidget</extends>
   <header location="global">controller.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
